x-pack/plugins/security_solution/public/common/components/sessions_viewer/index.tsx

```diff
@@ -38,7 +38,8 @@ export const defaultSessionsFilter: Required<Pick<Filter, 'meta' | 'query'>> = {
           bool: {
             // show sessions table results by filtering events where event.action is fork, exec, or end
             should: [
-              { term: { [EVENT_ACTION]: 'exec' } },
+              { term: { [EVENT_ACTION]: 'exec' } }, // exec event.action is used by Endpoint and Cloud Defend
+              { term: { [EVENT_ACTION]: 'executed' } }, // executed event.action is used by auditbeat
               { term: { [EVENT_ACTION]: 'fork' } },
               { term: { [EVENT_ACTION]: 'end' } },
```

Comment:
it would also be nice to define a structure for the event actions and use it everywhere without the need to mention that `executed` is for `auditbeat`, smth like 
```
const SESSEION_VIEWER_EVENT_ACTIONS = {
  EXEC: 'exec',
  EXEC_AUDITBEAT: 'executed',
  FORK: 'fork',
  END: 'end'
}
```
not a blocker for the PR I think, but I expect confusion in the future with this duality of `exec`

Reply:
@Omolola-Akinleye actually I see that you already defined the constants suggested by Paulo, let's export them and use in the filters as well. Plus, you can ignore my prev comment with the structure, but I would suggest renaming `EVENT_ACTION_EXECUTED` into `EVENT_ACTION_EXEC_AUDITBEAT`. wdyt?

Reply:
up to you! My concern is that you already needed to add the comment about it in two places, the name could have made the logic behind it clear from the name. And we can always change it in the future for any new requirements, we will never be 100% that now we cover every possible case :)